src/main/java/org/junit/runner/manipulation/GenericOrdering.java

```diff
@@ -0,0 +1,36 @@
+package org.junit.runner.manipulation;
+
+import java.util.Collection;
+import java.util.List;
+
+import org.junit.runner.Description;
+
+/**
+ * An {@link Ordering} that is not a {@link Sorter}.
+ *
+ * @since 4.13
+ */
+public final class GenericOrdering extends Ordering {
```

Comment:
@dsaff I would rather not call it `DelegatingOrdering`. It is specifically an `Ordering` that is known not to be a `Sorter`, and some of the core code depends on that. I'm open to other names and improved doc, of course.